docs/Materials.md.html

```diff
@@ -527,25 +527,32 @@
 
 This alternative lighting model exists to comply with legacy standards.
 
-Similar to cloth, it encompasses all the parameters previously defined for the standard
-lit mode except for `metallic` and `reflectance`, and adds a `sheenColor` parameter. The sheen
-color to directly control the _specular color_ of the material.
+This encompasses all the parameters previously defined for the standard lit mode except for
+_metallic_ and _reflectance_, and adds parameters for _specularColor_ and _glossiness_. The specular
+color is used to formulate the BRDF inputs as follows.
+
+```glsl
+pixel.diffuseColor = baseColor * (1.0 - max(specularColor.r, specularColor.g, specularColor.b));
```

Comment:
Don't put the math/code in this document, this is meant for users. Just explain that base color is diffuse and specular color is the specular color.

Reply:
Removed